src/index.js

```diff
@@ -0,0 +1,27 @@
+/* @flow */
+
+import SubscriptionServer from './SubscriptionServer';
```

Comment:
not using the export extensions?

Reply:
i never remember the syntax for what i want to do :P